### PR TITLE
[NEXUS-8658] Avoid double loading of users store

### PR DESCRIPTION
**Pull request by adreghiciu**
https://issues.sonatype.org/browse/NEXUS-8658

### CODE REVIEW

File: plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/controller/Users.js [hunk 1]
```diff
@@ -284,10 +284,14 @@ Ext.define('NX.coreui.controller.Users', {
...
-        callback: cb
+        callback: function(records, operation, success) {
+          if (Ext.isFunction(cb)) {
+            cb(records, operation, success);
+          }
+          me.reselect();
+        }
       });
     }
-    me.callParent();  // triggers transition between list/detail view
```
Comment (jdillon) [inline plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/controller/Users.js]: is loadStore() overriden?

Comment (adreghiciu) [inline plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/controller/Users.js]: Yes, to accommodate the user manager / user id filter

Comment (jdillon) [inline plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/controller/Users.js]: jsdoc should then use @override, and if you aren't going to call parent, probably explain why in a comment for clarity plz

Comment (adreghiciu) [inline plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/controller/Users.js]: Documnted

**Comment by kellyrob99** [comment]
+1

**Comment by jdillon** [comment]
+1

**Comment by mcculls** [comment]
+1

**Pull request merged**